x-pack/plugins/maps/public/actions/ui_actions.ts

```diff
@@ -24,6 +24,8 @@ export const SET_OPEN_TOC_DETAILS = 'SET_OPEN_TOC_DETAILS';
 export const SHOW_TOC_DETAILS = 'SHOW_TOC_DETAILS';
 export const HIDE_TOC_DETAILS = 'HIDE_TOC_DETAILS';
 export const SET_DRAW_MODE = 'SET_DRAW_MODE';
+export const SET_DELETED_FEATURES = 'UPDATE_DELETED_FEATURES';
```

Comment:
rename `SET_DELETED_FEATURES` to `PUSH_DELETED_FEATURE_ID`. Make sure value matches name
rename `REMOVE_DELETED_FEATURES` to `CLEAR_DELETED_FEATURE_IDS`